src/features/emails/components/EmailEditor/EmailSettings/PreviewTab.tsx

```diff
@@ -1,20 +1,29 @@
-import { FC } from 'react';
+import { FC, useEffect, useState } from 'react';
 import {
   Alert,
   Box,
   Button,
   CircularProgress,
+  TextField,
   Typography,
 } from '@mui/material';
+import z from 'zod';
+import { ErrorOutlined } from '@mui/icons-material';
 
 import messageIds from 'features/emails/l10n/messageIds';
 import { Msg } from 'core/i18n';
 import useCurrentUser from 'features/user/hooks/useCurrentUser';
 import useSendTestEmail from 'features/emails/hooks/useSendTestEmail';
 
+const emailAddressIsValid = (emailAddress: string): boolean =>
+  z.string().email().safeParse(emailAddress).success;
+
 const PreviewTab: FC = () => {
   const user = useCurrentUser();
   const { emailWasSent, isLoading, reset, sendTestEmail } = useSendTestEmail();
+  const [emailError, setEmailError] = useState(false);
+  const [destinationEmailAddress, setDestinationEmailAddress] = useState('');
+  useEffect(() => setDestinationEmailAddress(user?.email ?? ''), [user?.email]);
```

Comment:
I think this `useEffect()` should only reset the state if it hasn't already been touched by the user, i.e.:

```suggestion
  useEffect(() => {
    if (user && destinationEmailAddress == '') {
      setDestinationEmailAddress(user.email);
    }
  }, [user?.email]);
```

Please note that although github has a feature to commit this immediately via the web interface, I have not scrutinized my suggestion entirely, so I would recommend applying it manually and making sure formatting etc is ok (and that it works!) before committing.

Reply:
Even if I do not fully understand in which case the user state is being reset, this definetely is a good idea. I implemented your requested change in 99a4c86

Reply:
fixed the comparison to 3 equal signs instead of 2 (I guess this is better, right? 😅 )
-> force pushed to d4f7d95

Reply:
I actually only use strict equality (`===`) in some very special cases in typescript. In javascript I understand why people like it, but in typescript most of the benefits from strict equality are solved in other ways, and so I try to be more intentional about when I use it and when I don't.

I recently commented in more length on this topic here: https://github.com/zetkin/app.zetkin.org/pull/2426#discussion_r1887956255

Reply:
I did not had any strong opinion about this up this point. 
Somehow the habit of always using strict equality (in typescript as well as in javascript) was present.

So I guess I have learned something new today :) 
Thank you @richardolsson

Reply:
Just out of curiosity:
We could also use the truthyness here, right?
Since an empty string will be falsy?

Like this here:

```ts
if (user && !destinationEmailAddress) {
   // ...
}
```

Reply:
I'm happy to have been able to give you something today!

With regards to your suggestion, I actually wasn't sure that empty strings are falsy in javascript, but you're right, they are!